src/excercise3.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Created by ttn on 27/12/16.
+ *
+ * Create a component called "Header" which either displays the current logged in username (if a user is logged in) or a "Login" button (if no user is logged in) in the navbar. Use a hardcoded global variable "isUserLoggedIn" to check if user is logged in or not.Create a component called "Header" which either displays the current logged in username (if a user is logged in) or a "Login" button (if no user is logged in) in the navbar.
+ * Use a hardcoded global variable "isUserLoggedIn" to check if user is logged in or not.
+ */
+import React from 'react';
+import {render} from 'react-dom';
+
+export default class Header extends React.Component {
+
+  constructor() {
```

Comment:
pass props as a variable to super as a best practice to avoid having this.props as undefined in constructor
```constructor(props) { super(props) }```